teku/src/main/java/tech/pegasys/teku/cli/options/ExecutionEngineOptions.java

```diff
@@ -43,7 +43,7 @@
   private Version executionEngineVersion = Version.DEFAULT_VERSION;
 
   @Option(
-      names = {"--Xee-payload-builders"},
+      names = {"--ee-payload-builders"},
```

Comment:
And payload builders is probably not ready to expose yet as the MEV stuff is still in progress.

`--ee-endpoint` and `--ee-jwt-secret` are good, though we should rename `--ee-jwt-secret` to `--ee-jwt-secret-file` to be consistent with our CLI naming